xls/dslx/tests/errors/error_modules_test.py

```diff
@@ -1188,6 +1188,17 @@ def test_gh_1473(self):
         ' which is not present in the parametric environment', stderr
     )
 
+  def test_nominal_type_mismatch(self):
+    # When the types are structurally identical and also have the same name by
+    # virtue of being defined in two different modules, we fully qualify the
+    # module where the struct came from in the error message.
+    stderr = self._run('xls/dslx/tests/errors/nominal_type_mismatch.x')
+    self.assertIn(
+        'Type mismatch:\n'
+        '   xls/dslx/tests/errors/mod_simple_struct.x:MyStruct {}\n'
+        'vs xls/dslx/tests/errors/mod_simple_struct_duplicate.x:MyStruct {}\n', stderr
```

Comment:
and this
might get a bit long if someone was to prepend some prefix, say, `third_party/` to `xls/dslx/` :)

Are there Python-ways to break these ? Backslash or other concatenation ?

Reply:
Ah good point, let me change it to assertMatchesRegexp so prefixes can be there.